homeassistant/components/bmw_connected_drive/__init__.py

```diff
@@ -26,34 +26,17 @@
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers.entity import DeviceInfo, Entity
 from homeassistant.helpers.event import track_utc_time_change
-from homeassistant.helpers.typing import ConfigType
 from homeassistant.util import slugify
 import homeassistant.util.dt as dt_util
 
-from .const import (
-    ATTRIBUTION,
-    CONF_ACCOUNT,
-    CONF_ALLOWED_REGIONS,
-    CONF_READ_ONLY,
-    DATA_ENTRIES,
-    DATA_HASS_CONFIG,
-)
+from .const import ATTRIBUTION, CONF_ACCOUNT, CONF_READ_ONLY, DATA_ENTRIES
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = "bmw_connected_drive"
 ATTR_VIN = "vin"
 
-ACCOUNT_SCHEMA = vol.Schema(
-    {
-        vol.Required(CONF_USERNAME): cv.string,
-        vol.Required(CONF_PASSWORD): cv.string,
-        vol.Required(CONF_REGION): vol.In(CONF_ALLOWED_REGIONS),
-        vol.Optional(CONF_READ_ONLY): cv.boolean,
-    }
-)
-
-CONFIG_SCHEMA = vol.Schema({DOMAIN: {cv.string: ACCOUNT_SCHEMA}}, extra=vol.ALLOW_EXTRA)
+CONFIG_SCHEMA = cv.deprecated(DOMAIN, raise_if_present=False)
```

Comment:
It's not only deprecated. It won't work any more. Did you test passing in the old config with this schema? Does it still pass config validation?
```suggestion
CONFIG_SCHEMA = cv.removed(DOMAIN, raise_if_present=False)
```

Reply:
Oops... Sorry, my bad!

When using `cv.removed`, the following error is logged:
```log
2022-02-22 18:43:40 ERROR (MainThread) [homeassistant.components.bmw_connected_drive] The 'bmw_connected_drive' option near /home/richard/home-assistant-core/config/configuration.yaml:2 has been removed, please remove it from your configuration
```

The component is still loaded normally via config flow.

Reply:
And that is the one that is now logged with the latest push (ef7a3ad) :)